### PR TITLE
Adding IBM Cloud installation docs

### PR DESCRIPTION
**Pull request by iancrowther**
**Description**

Draft PR for IBM docs

Changes proposed in this pull request:

- IBM Cloud CLI steps for installer

**Related issue(s)**

`See also #5961`

### CODE REVIEW

**Comment by mmitoraj** [comment]
Thank for your contribution! I will gladly review the document. Could you first rebase as your branch does not seem to be created from the most up-to-date master?

File: docs/kyma/04-03-cluster-installation.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+4. Add your account as the cluster administrator:
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value account)
+    ```
+  
+  </details>
+
+  <details>
```
Comment (mmitoraj) [inline docs/kyma/04-03-cluster-installation.md]: To be consistent with the  **Prerequisites** section, I would add the IBM Cloud tab in between the AKS and Gardener (line 195).

File: docs/kyma/04-04-cluster-installation.md [hunk 2]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
...
+    ibmcloud ks cluster create classic --zone ${CLUSTER_ZONE} --machine-type b3c.4x16 --workers 3 --name ${CLUSTER_NAME} --public-service-endpoint
+    ```
+
+3. Configure kubectl to use your new cluster. Run:
+
+    ```bash
+    ibmcloud ks cluster config --cluster ${CLUSTERN_NAME}
+    ```
+
+    >**TIP:** Copy and paste the export command that is displayed in your terminal to set the KUBECONFIG environment variable.
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
4. Copy, paste, and run the export command that is displayed in your terminal to set the `KUBECONFIG` environment variable.
```

File: docs/kyma/04-04-cluster-installation.md [hunk 3]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
...
+
+4. Add your account as the cluster administrator:
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
5. Add the cluster administrator role to your user account:
```
Is it really an account you're adding or a role? What do you think about the suggested version?

File: docs/kyma/04-04-cluster-installation.md [hunk 4]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
 
 >**CAUTION:** If you define your own Kubernetes jobs on the AKS cluster, follow [this](/components/service-mesh/#troubleshooting-kubernetes-jobs-fail-on-aks) troubleshooting guide to avoid jobs running endlessly on AKS deployments of Kyma.
 
+  </details>
+  <details>
+  <summary label="IBM Cloud">
+  IBM Cloud
+  </summary>
+  
+1. Select a name for your cluster. Export the cluster name and the zone of type ```dc``` you want to deploy to as environment variables. Run:
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
1. Select a name for your cluster. Export the cluster name, the zone of type `dc` you want to deploy to, and your IBM Cloud account email address as environment variables. Run:
```

File: docs/kyma/04-04-cluster-installation.md [hunk 5]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
...
+
+    ```bash
+    export CLUSTER_NAME={CLUSTER_NAME_YOU_WANT}
+    export CLUSTER_ZONE={ZONE_TO_DEPLOY_TO}
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
    export CLUSTER_ZONE={ZONE_TO_DEPLOY_TO}
    export IBM_EMAIL={YOUR_IBM_CLOUD_ACCOUNT_EMAIL}
```

File: docs/kyma/04-04-cluster-installation.md [hunk 6]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
...
+    ```
+
+    >**TIP:** Run: ```ibmcloud ks supported-locations``` to list available zones. 
+
+2. Create a cluster in the defined zone. Run:
+
+    ```bash
+    ibmcloud ks cluster create classic --zone ${CLUSTER_ZONE} --machine-type b3c.4x16 --workers 3 --name ${CLUSTER_NAME} --public-service-endpoint
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
    ibmcloud ks cluster create classic --zone $CLUSTER_ZONE --machine-type b3c.4x16 --workers 3 --name $CLUSTER_NAME --public-service-endpoint
```

File: docs/kyma/04-04-cluster-installation.md [hunk 7]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
...
+    ```
+
+3. Configure kubectl to use your new cluster. Run:
+
+    ```bash
+    ibmcloud ks cluster config --cluster ${CLUSTERN_NAME}
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
    ibmcloud ks cluster config --cluster $CLUSTER_NAME
```

File: docs/kyma/04-04-cluster-installation.md [hunk 8]
```diff
@@ -150,6 +162,41 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
...
+    ```
+
+    >**TIP:** Copy and paste the export command that is displayed in your terminal to set the KUBECONFIG environment variable.
+
+4. Add your account as the cluster administrator:
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$[Your IBM Cloud account email]
```
Comment (mmitoraj) [inline docs/kyma/04-04-cluster-installation.md]:
```suggestion
    kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$IBM_EMAIL
```

**Comment by mmitoraj** [comment]
For now, merging is blocked by https://github.com/kyma-project/kyma/issues/6451.

**Comment by iancrowther** [comment]
blocked by https://github.com/kyma-project/kyma/issues/6451

**Comment by kazydek** [comment]
Hi @iancrowther! @mmitoraj  is on maternity leave now so I would like to help you with your contribution hoping that you’ve not lost your enthusiasm about it ;) I’ve consulted it with @michal-hudy (our PO) and he suggested adding a NOTE to the instruction informing that you should install Kyma without the logging module. I can prepare this NOTE for you and go through the PR again to make sure all is corrected. Unfortunately, we don’t have an account now on which we could test this instruction. The question is if you could test it, install Kyma without the logging module, and let me know if you encountered any issues? Let me know how you want me to help you with this PR - I would gladly help you with this PR or prepare a new PR if you like, with your changes since I can see that the doc you were updating has changed a bit in the meantime (some time has passed…)

**Comment by kazydek** [comment]
As a follow-up, I created a separate PR:
- without `04-03-cluster-installation.md`
- with CAUTION informing to disable the Logging component before installing Kyma

See the new PR [here](https://github.com/kyma-project/kyma/pull/5964).

**Pull request closed without merging**